src/StorageVolume.cpp

```diff
@@ -56,6 +56,11 @@ int CJNIStorageVolume::getDescriptionId()
   }
 }
 
+CJNIFile CJNIStorageVolume::getDirectory()
+{
+  return call_method<jhobject>(m_object, "getDirectory", "()Ljava/io/File;");
```

Comment:
Does this need to be gated by an sdk version check?
introduced in API 30 - https://developer.android.com/reference/android/os/storage/StorageVolume#getDirectory()

Reply:
Thanks, I just realised that I didn't check the level in the Kodi PR. I will modify it to keep the previous implementation if level is less than 30.